libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -128,6 +128,13 @@ float AP_RSSI::read_receiver_rssi()
         case RssiType::RSSI_RC_CHANNEL_VALUE :
             receiver_rssi = read_channel_rssi();
             break;
+        case RssiType::RSSI_RECEIVER : {
+            int16_t rssi = hal.rcin->get_rssi();
+            if (rssi != -1) {
+                receiver_rssi = rssi / 255.0;
+            }
```

Comment:
I'm happy to add range parameters, but it shouldn't be limited to the "receiver RSSI".
Right now we have input scaling parameters for voltage and for PWM channel RSSI. Those are needed as the scale of those is not well defined or under our control. So we have to have those input scaling parameters.
For receiver based RSSI we don't need input scaling parameters as the range is defined by the protocol with the px4 orb.
We could however add output scaling, in which case it should apply to all RSSI types.